src/material/core/tokens/_m3-system.scss

```diff
@@ -50,6 +111,13 @@ $_system-level-prefix: sys;
     definitions.md-sys-color-values-dark($ref),
     definitions.md-sys-color-values-light($ref));
 
+  // Manually insert a subset of palette values that are used directly by components
```

Comment:
Why is this different than other components? Shouldn't they reference the system variables? Or will they eventually?

Reply:
I wish =\ seems the spec specifically says the palette values are the source here, which is weird. There's even a `scrim` value in the system that I would expect the sidenav to use. We can change this later if we want to deviate from the spec